MapboxCoreNavigation/RouteProgress.swift

```diff
@@ -131,6 +135,30 @@ open class RouteProgress: NSObject {
         self.legIndex = legIndex
         super.init()
         currentLegProgress = RouteLegProgress(leg: currentLeg, stepIndex: 0, alertLevel: alertLevel)
+        
+        var coordinateIndex = 0
```

Comment:
`maneuverCoordinateIndex` would be more descriptive. A documentation comment like this would help as well (you can give local variables documentation comments too):

> An index into the route’s `coordinates` and `congestionTravelTimesSegmentsByStep` that corresponds to a step’s maneuver location.